api/src/controllers/kots/KotsAPI.ts

```diff
@@ -752,7 +756,21 @@ export async function syncLicense(stores, app, airgapLicense) {
   const bundlePacker = new TarballPacker();
   const tarGzBuffer: Buffer = await bundlePacker.packFiles(files);
 
-  await uploadUpdate(stores, app.slug, tarGzBuffer, "License Update");
+  const tmpDir = tmp.dirSync();
+  try {
+    const inputArchivePath = path.join(tmpDir.name, "input.tar.gz");
+    const outputArchive = path.join(tmpDir.name, "output.tar.gz");
+    fs.writeFileSync(inputArchivePath, tarGzBuffer);
+
+    const downstreams = await stores.kotsAppStore.listDownstreamsForApp(app.id);
+    const registrySettings = await stores.kotsAppStore.getAppRegistryDetails(app.id);
```

Comment:
is this just for airgap?

Reply:
@salahalsaleh1993 no this is for online only.  You can have a private registry with online installs.

Reply:
ok, then won't this be called in the airgap scenario as well?

Reply:
I do believe that airgap installs can hit this code. you can upload a new license from the license page. and it will hit this code (from what i remember).